README.md

```diff
@@ -177,83 +181,95 @@ Now each of the above list must be converted to bytearrays, i.e. `Uint8Array` an
 Before messages can be signed, 2 things are needed:
 
 - **Signature parameters**: Public values, that can be created by anyone but must be known to the signer and verifier to sign and verify respectively. To create them, the number of messages (attributes) being signed must be known and the size of the parameters increases with the number. In the above example, number of attributes is 4. These parameters can be generated randomly or deterministically by using a publicly known label. It is advised to use the latter as it allows for extending/shrinking the same parameters when number of messages change.
-- **Keypair**: To create and verify BBS+ signature, the signer (issuer in case of a credential) needs to create a secret key to sign, public key to verify. 
+- **Keypair**: To create and verify BBS signature, the signer (issuer in case of a credential) needs to create a secret key to sign, public key to verify. 
```

Comment:
Same as above - but it seems that most other usages are `BBS signatures`, so I'd stick with the plural throughout the document, except where it doesn't make sense...

Reply:
Either "verify a BBS signature" or "verify BBS signatures".